npm/npm-shrinkwrap.json

```diff
@@ -1,211 +1,8 @@
 {
   "name": "@cloudflare/wrangler",
-  "version": "1.18.0",
-  "lockfileVersion": 2,
+  "version": "1.19.0",
+  "lockfileVersion": 1,
```

Comment:
I think I found why it changed so much - do you have an old version of npm maybe?

Reply:
Fix me